pymc3/sampling.py

```diff
@@ -265,20 +265,26 @@ def _choose_backend(trace, chain, shortcuts=None, **kwds):
     except KeyError:
         raise ValueError('Argument `trace` is invalid.')
 
+def _make_parallel(arg, njobs):
+    if not shape(arg):
```

Comment:
`if np.shape == (0, )`? But I agree that it's probably fine in this case so lets not block on it either way.